mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioNavigator.tsx

```diff
@@ -190,6 +192,20 @@ export function PortfolioNavigator (): JSX.Element {
         }}
       />
 
+      <PortfolioStack.Screen
+        component={ReceiveDTokenScreen}
+        name='ReceiveDTokenScreen'
+        options={{
+          headerTitle: () => (
+            <HeaderTitle
+              text={translate('screens/ReceiveDTokenScreen', 'Bitcoin Deposit')}
```

Comment:
DToken or Bitcoin?

Reply:
Screen Design just says Bitcoin Deposit. Component will grow in future, therefore future ready naming. Will change when more external Crypto's are added.